conda-recipe/bld.bat

```diff
@@ -4,6 +4,9 @@ if errorlevel 1 (
     exit /b 1
 )
 
+git clone -b v1.2.3 https://github.com/oneapi-src/level-zero.git
```

Comment:
Please make comment that tag version is not very important. Headers are backward compatible.